### ISSUE TITLE
export transformation script

### ISSUE THREAD

**Issue by ivozandhuis** [netwerk-digitaal-erfgoed/LDWizard#44]
Is Cow the right name for the script-language? I'd say "csvw" is the right format and cow is the tool?

**Comment by rlzijdeman** [comment]
no, and neither is csvw. I'll get back with a more satisfying answer.

________________________________
